snowflake-resource-optimization/README.md

```diff
@@ -0,0 +1,72 @@
+# Snowflake Resource Optimization: Setup & Configuration
```

Comment:
If we shorten this name, it could make it easier to remember and say. 

Maybe: "Snowflake Resource Optimization Dashboard"

Any thoughts?

(The exact name could depend on whether this app covers just the Setup & Configuration piece, or is meant to be broader than that. I wasn't sure about the scope.)

Reply:
(we can brainstorm live at our sync tomorrow!)

Reply:
decided on Snowflake Setup & Configuration Optimization

Reply:
I'd disagree. This is 1 of a 4 part series. I'd prefer to keep it named exactly the same as Snowflake does
`Resource Optimization: Setup & Configuration`

Reply:
keeping original name